app/controllers/stories_controller.rb

```diff
@@ -5,6 +5,10 @@ class StoriesController < ApplicationController
   # POST /projects/:project_id/stories
   def create
     @story = @project.stories.create!(story_params)
+    project_position = ProjectPosition.find_by(story_id: @story.id)
+    if project_position.present?
+      project_position.move_to_bottom
+    end
```

Comment:
@anjakammer if I remove this, the story gets inserted at the top, therefore I figured it's necessary

Reply:
It should go to the model.
Here app/models/story.rb:54

```
def create_project_position
    project_position = ProjectPosition.create(
        :project_id => self.project_id, :story_id => self.id
    )
    project_position.set_list_position(0)
    project_position.save!
  end
```

There should be the `project_position.move_to_bottom` instead of `project_position.set_list_position(0)`

Reply:
makes sense, thanks! replacing it directly doesn't work, or there will be an error that the position isn't set, but putting it below works just fine.